rest_framework/renderers.py

```diff
@@ -260,88 +268,93 @@ class HTMLFormRenderer(BaseRenderer):
     base_template = 'form.html'
 
     default_style = ClassLookupDict({
-        serializers.Field: {
+        Field: {
```

Comment:
Again, would much rather keep the `serializers.<SomeField>` style throughout. It's an important bit of namespacing, and we've been very strongly consistent about it's use and recommending it to others. If the linting tool is going to force this to change, then I don't see us accepting that aspect of the linting.